python-avd/pyavd/_eos_designs/schema/eos_designs.schema.yml

```diff
@@ -3180,25 +3180,14 @@ keys:
 
             '
   ptp:
+    deprecation:
```

Comment:
please add a test for ptp in `eos_designs_unit_tests_deprecated_vars` moelcule

Reply:
Added.